spec/javascripts/helper/reactionVariationsHelpers.js

```diff
@@ -21,4 +21,21 @@ async function setUpReaction() {
   return reaction;
 }
 
-export { setUpMaterial, setUpReaction };
+function getColumnGroupChild(columnDefinitions, groupID, fieldID) {
```

Comment:
Can be used in ReactionVariationsUtils.js updateColumnDefinitions( )  to easy method

Reply:
I realized that using `getColumnGroupChild()` in `updateColumnDefinitions()` isn't convenient since I don't have access to `getColumnGroupChild()`'s `groupID` and `fieldID` parameters in the context of `updateColumnDefinitions()`. Also, `getColumnGroupChild()` is a test helper that I'd rather not use in the main code.